deployment/src/main/java/io/quarkus/jgit/deployment/JGitBuildTimeConfig.java

```diff
@@ -48,6 +48,12 @@ interface DevService {
         @WithDefault("quarkus")
         String adminPassword();
 
+        /**
+         * The organization to be created when the Dev Service starts.
+         */
+        @WithDefault("dev")
+        String organization();
```

Comment:
```suggestion
        Optional<List<String>> organizations();
```

Reply:
Actually instead of introducing a new org, what if the `repositories()` could specify them? Eg. `quarkus.jgit.devservices.repositories=dev/foo,bar` would create an org named `dev` and a repository `foo` in there and a `bar` in the user organization

Reply:
This parameter is useless now that the organization name is included in the repository name

Reply:
Agree. Updated the pull request so that:
- user is able to specify org repositories using: <org>/<repo> pattern.
- user is able to specify user repositories using: <repo> pattern.
- user is able to specify organizations without repositories.

Reply:
Ah, organizations without repositories, good call, haven't thought about that

Reply:
Can you also update the `repositories()` Javadoc with this explanation?